Packages/com.unity.inputsystem/InputSystem/Editor/InputAssetEditorUtils.cs

```diff
@@ -97,8 +97,11 @@ public static void DrawMakeActiveGui<T>(T current, T target, string targetName,
             if (!string.IsNullOrEmpty(currentlyActiveAssetsPath))
                 currentlyActiveAssetsPath = $" The actions currently assigned as the {entity} are: {currentlyActiveAssetsPath}. ";
             EditorGUILayout.HelpBox($"These actions are not assigned as the {entity} for the Input System. {currentlyActiveAssetsPath??""}", MessageType.Warning);
+            GUI.enabled = allowAssignActive;
             if (GUILayout.Button($"Assign as the {entity}", EditorStyles.miniButton))
                 apply(target);
+            if (!allowAssignActive)
```

Comment:
Minor: I don't think you really need this test here. Its potentially slightly more performant if the GUI.enabled assignment does more work in the background, but I'm not it does.
I usually store the previous GUI.enabled state in a variable and restore it, but TBH it will be enabled above so you could just set it.

Reply:
Will remove the check